packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -53,17 +148,69 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// [duration].
   final Widget child;
 
+  /// The duration over which to perform the cross fade using [FadeTransition].
```

Comment:
The duration of the transition from the old [child] value to the new one.

Reply:
Done.